test/unit/Reflection/ReflectionMethodTest.php

```diff
@@ -128,4 +131,44 @@ public function testMethodNameWithNamespace()
         $this->assertSame('', $methodInfo->getNamespaceName());
         $this->assertSame('someMethod', $methodInfo->getShortName());
     }
+
+    public function modifierProvider()
+    {
+        return [
+            ['publicMethod', 256, ['public']],
```

Comment:
Consider using internal reflection flags/constants here (in order to guarantee compat)